src/app/contextBroker/orionRestServices.cpp

```diff
@@ -510,7 +510,7 @@ static RestService badVerbV[] =
   { InvalidRequest,                                2, { "ngsi9",   "*"                                                                 }, badNgsi9Request           },
   { InvalidRequest,                                2, { "ngsi10",  "*"                                                                 }, badNgsi10Request          },
   { InvalidRequest,                                0, { "*", "*", "*", "*", "*", "*"                                                   }, badRequest                },
-  { InvalidRequest,                                0, {                                                                                }, NULL                      },
```

Comment:
Does the refactor and changes done in this PR has any impact in Developers Manual?

Reply:
No, we didn't get down to this level of details in the manual, fortunately ...